src/test/java/winstone/testCase/HttpConnectorTest.java

```diff
@@ -64,58 +63,23 @@ public void testSimpleConnection() throws IOException, SAXException,
         Launcher winstone = new Launcher(args);
 
         // Check for a simple connection
-        WebConversation wc = new WebConversation();
-        WebRequest wreq = new GetMethodWebRequest(
-                "http://localhost:10003/examples/CountRequestsServlet");
-        WebResponse wresp = wc.getResponse(wreq);
-        InputStream content = wresp.getInputStream();
-        assertTrue("Loading CountRequestsServlet", content.available() > 0);
-        content.close();
-        winstone.shutdown();
-        Thread.sleep(500);
-    }
-
-    /**
-     * Test the keep alive case
-     */
-    public void testKeepAliveConnection() throws IOException,
-            InterruptedException, SAXException {
-        // Initialise container
-        Map<String,String> args = new HashMap<>();
-        args.put("webroot", "target/testwebapp");
-        args.put("prefix", "/examples");
-        args.put("httpPort", "10004");
-        args.put("ajp13Port", "-1");
-        args.put("controlPort", "-1");
-        args.put("debug", "8");
-        args.put("logThrowingLineNo", "true");
-        Logger.init(Logger.FULL_DEBUG, System.out, Charset.defaultCharset(), true);
-        Launcher winstone = new Launcher(args);
-
-        // Check for a simple connection
-        WebConversation wc = new WebConversation();
-        WebRequest wreq = new GetMethodWebRequest(
-                "http://localhost:10004/examples/CountRequestsServlet");
-        WebResponse wresp1 = wc.getResponse(wreq);
-        WebImage[] img = wresp1.getImages();
-        for (WebImage anImg : img) wc.getResponse(anImg.getRequest());
-        // Thread.sleep(2000);
-        // WebResponse wresp2 = wc.getResponse(wreq);
-        // Thread.sleep(2000);
-        //WebResponse wresp3 = wc.getResponse(wreq);
-        InputStream content = wresp1.getInputStream();
-        assertTrue("Loading CountRequestsServlet + child images", content
-                .available() > 0);
-        content.close();
+        HttpRequest request =
+                HttpRequest.newBuilder(new URI("http://localhost:10003/examples/CountRequestsServlet")).GET().build();
```

Comment:
while you're doing some cleaning. maybe using a random port here would be good as well.
same as done in `ArgumentsRealmTest`

Reply:
Not interested